File: week-2/insertion.py
def insertionSort(lst):
    for i in range(len(lst) - 1):
        j = i + 1
        while j < len(lst) and lst[j] < lst[i]:
            lst[i], lst[j] = lst[j], lst[i]
            j += 1
    print(lst)

    return lst

insertionSort([2,0,2,1,1,0])

File: week-2/insertion.js
function insertionSort(ar) {
    for(let i = 1; i <= ar.length - 1; i++) {
        for(let j = i; j > 0; j--) {
            if(ar[j - 1] > ar[j]) {
                let temp = ar[j - 1];
                ar[j - 1] = ar[j];
                ar[j] = temp;
            }
        console.log(ar);
        }
    }
}

console.log(insertionSort([4, 3, 2, 1]));
